pyanaconda/kickstart.py

```diff
@@ -293,25 +295,32 @@ def parse(self, args):
         log.warning("Command %s will be parsed in DBus module.", self.currentCmd)
 
 
-class Authselect(commands.authselect.F28_Authselect):
+class Authselect(RemovedCommand):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.packages = []
 
+    def __str__(self):
+        return ""  # It is generated by the SELinux command.
```

Comment:
I think some more backstory for this is needed.

Reply:
Fixed.